SupremacyCore/Combat/AutomatedCombatEngine.cs

```diff
@@ -201,15 +203,15 @@ protected override void ResolveCombatRoundCore()
                 {
                     // UPDATE X 25 june 2019 added if == 999 & warlike then choose a random target. Also DiplomaticReport needs to change to traits, but currently everyone has trait = compassion
                     // deleted: (empiresInBattle[q, 1] == 777 || empiresInBattle[q, 1] == 999) &&
-                    if ( (civi.DiplomacyReport.Contains("Warlike") || civi.DiplomacyReport.Contains("Hostile")))
+                    if ((civi.DiplomacyReport.Contains("Warlike") || civi.DiplomacyReport.Contains("Hostile")))
```

Comment:
It works I guess. Personally I wouldn't have to the traits contained in the Diplomacy Report however - it's a text field for giving an overview of a civilization. A better way would be to have an array or list of traits